spa/src/components/routes/RouterSetup.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
```suggestion
import { Suspense } from 'react';
```
(since that's our usual pattern)